features/angle.yml

```diff
@@ -0,0 +1,11 @@
+name: angle
+description: The `angle` CSS type represents a portion of a circle, expressed in degrees (`deg`), gradians (`grad`), radians (`rad`), or turns (`turn`). The `angle-percentage` CSS type represents an angle or percentage value.
+spec: https://drafts.csswg.org/css-values-4/#angles
+group: css
```

Comment:
Should we have a CSS types subgroup?

Reply:
It's definitely getting big enough. Would that work for typed OM as well?